index.xml

```diff
@@ -374,6 +376,41 @@ tampered with in transit.
    </t>
 
   </section>
+
+  <section anchor="date-header" title="Date Header Usage">
+
+   <t>
+This specification makes use of the Date header in illustrations,
+in the default list of headers to sign where none is explicitly provided, and
+as a one of headers to include as a minimum in the signature in implementations.
+   </t>
+
+   <t>
+The meaning and construction of the Date header is the one defined in
+<xref target="RFC7231">RFC 7231</xref>, <eref
+target="https://tools.ietf.org/html/rfc7231#section-7.1.1.2">
+Section 7.1.1.2</eref>.
+Note that the definition used defines the Date header as required in all
+Responses, but OPTIONAL in Requests unless it will
```

Comment:
No need to capitalize Responses or Requests... keep it lowercase as it's a regular term of art in most RFCs (unless I'm misunderstanding something).

Reply:
True, I'm just in the habit when using formal terms in correspondence.
Resolved 1d86bf7eab554a73fb2008abfd9f2cb7318a228c